src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -174,6 +174,7 @@ def upgrade_status(self) -> orchestrator.UpgradeStatusSpec:
                 r.message = 'Error: ' + self.upgrade_state.error
             elif self.upgrade_state.paused:
                 r.message = 'Upgrade paused'
+                r.is_paused = True
```

Comment:
maybe I missed it, but when is this set to False (unpaused)?

Reply:
it's set up so that the default value is false and we make a fresh `UpgradeStatusSpec` every time this is called so it will be false unless `r.is_paused = True` is explicitly run

Reply:
true, I have no idea. @adk3798 ?

Reply:
```bash
[ceph: root@upgrade-status-1 /]# ceph orch upgrade start localhost:5000/guits:v2
Initiating upgrade to localhost:5000/guits:v2
[ceph: root@upgrade-status-1 /]# ceph orch upgrade status
{
    "target_image": "localhost:5000/guits@sha256:921f864070a6b059e860fcc249e1b6997c1280e88de54840cd5f9ddba0c87e5a",
    "in_progress": true,
    "which": "Upgrading all daemon types on all hosts",
    "services_complete": [],
    "progress": "0/4 daemons upgraded",
    "message": "Currently upgrading mgr daemons",
    "is_paused": false
}
[ceph: root@upgrade-status-1 /]# ceph orch upgrade pause
Paused upgrade to localhost:5000/guits@sha256:921f864070a6b059e860fcc249e1b6997c1280e88de54840cd5f9ddba0c87e5a
[ceph: root@upgrade-status-1 /]# ceph orch upgrade status
{
    "target_image": "localhost:5000/guits@sha256:921f864070a6b059e860fcc249e1b6997c1280e88de54840cd5f9ddba0c87e5a",
    "in_progress": true,
    "which": "Upgrading all daemon types on all hosts",
    "services_complete": [],
    "progress": "1/4 daemons upgraded",
    "message": "Upgrade paused",
    "is_paused": true
}
[ceph: root@upgrade-status-1 /]# ceph orch upgrade resume
Resumed upgrade to localhost:5000/guits@sha256:921f864070a6b059e860fcc249e1b6997c1280e88de54840cd5f9ddba0c87e5a
[ceph: root@upgrade-status-1 /]# ceph orch upgrade status
{
    "target_image": "localhost:5000/guits@sha256:921f864070a6b059e860fcc249e1b6997c1280e88de54840cd5f9ddba0c87e5a",
    "in_progress": true,
    "which": "Upgrading all daemon types on all hosts",
    "services_complete": [],
    "progress": "1/4 daemons upgraded",
    "message": "",
    "is_paused": false
}
```